ow_faults/src/FaultInjector.cpp

```diff
@@ -272,14 +266,20 @@ void FaultInjector::jointStateCb(const sensor_msgs::JointStateConstPtr& msg)
     m_system_faults_bitset &= ~isArmExecutionError;
   }
 
-  if (!m_faults.camera_left_trigger_failure) {// if fault
+  if (m_cam_fault){
+    m_system_faults_bitset |= isCamExecutionError;
+    setComponentFaultsMessage(camera_faults_msg, hardwareFault);
+  }
+  else {
```

Comment:
be consistent with your style
```suggestion
  } else {
```